curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f43d341a0dd17120cdb08.md

```diff
@@ -7,20 +7,20 @@ dashedName: step-17
 
 # --description--
 
-You will need to convert uppercase characters to lowercase and add an underscore before them. 
+You will need to convert uppercase characters to lowercase and add an underscore before them. Before proceeding, join all the elements of the list `snake_cased_char_list` into a single string using an empty string `''` as the separator.
 
-Within the empty list, implement an `if` statement to check whether the character is uppercase. If it is, append `'_' + char.lower()` to the list.
+The using the return statement, return an empty string `''` joined with `snake_cased_char_list` as the argument, and strip any leading or trailing underscores from the result.
```

Comment:
Actually, I think, we could add join and strip at later steps and here just return `snake_cased_char_list`, so that it can be seen the actual output of the list comprehension